Octokit/Clients/PullRequestsClient.cs

```diff
@@ -36,6 +37,14 @@ public PullRequestsClient(IApiConnection apiConnection) : base(apiConnection)
         public IPullRequestReviewRequestsClient ReviewRequest { get; set; }
 
         /// <summary>
+        /// Access GitHub's Pull Requests Review API.
+        /// </summary>
+        /// <remarks>
+        /// Refer to the API documentation for more information: https://developer.github.com/v3/pulls/reviews/
+        /// </remarks>
+        public IPullRequestReviewClient PullRequestReview { get; set; }
```

Comment:
`Review` rather than `PullRequestReview` and ideally order it above the other 2 to match API doc layout (my OCD tendency sorry!)